src/Entity/Transformer.php

```diff
@@ -242,31 +251,9 @@ public function getChildren(PageInterface $page)
     }
 
     /**
-     * @return array
-     */
-    protected function fixPageContent(array $content)
```

Comment:
1 - why do you need to remove this, just to cover with tests?
2 - as this class is not final, and if someone is using this method, it will break!

Reply:
Indeed, missed that ; I thought this was private/final.

We have to keep this on 3.x and add a NEXT_MAJOR to remove it in 4.x

Reply:
curious, i thought they where only private in 4.x

might be a change in between, i can just add them back, and to be removed later in 4.x